r/R/dplyr-funcs-datetime.R

```diff
@@ -115,6 +115,23 @@ register_bindings_datetime <- function() {
       return(Expression$create("strftime", x, options = list(format = format, locale = locale)))
     }
 
+    if (call_binding("is.integer", x)) {
+      if (inherits(x, "Expression")) {
+        call_binding(
+          "if_else",
+          call_binding_agg("all", call_binding("between", x, 1, 12))$data,
+          x,
+          abort("bla: Values are not in 1:12")
+        )
```

Comment:
I suspect in this part of the branch what you'll need to do is (re) assign the `call_binding` to `x` or return it directly. (thought see the comment below that this probably needs to happen above or we'll need to take into account that `label` argument.